.github/workflows/e2e-component-tests-embedding-sdk-cross-version.yml

```diff
@@ -165,8 +165,9 @@ jobs:
 
       - name: Move tests from the release commit into the repo
         run: |
-          rm -rf ./e2e/support/helpers/*
-          mv ./sdk-release-e2e/e2e/support/helpers/* ./e2e/support/helpers
+          rm -rf ./e2e/support/*
```

Comment:
I removed the line that keeps `component-webpack.config.js` because by the time we cut 54. The webpack config would already be fixed since it's already fixed now (if no further fixes are needed)